config/common_test.go

```diff
@@ -39,6 +39,7 @@ func init() {
 			DefaultClientLocale: sToP("en"),
 		},
 	}
+	minimalConfig.SetDefaults()
```

Comment:
It's not super obvious why this is required? In particular, I recall `minimalConfig` supposedly representing the minimum number of "bits" defined to "pass", but with the defaults set, we no longer achieve that minimum.

Reply:
Basically it's because some tests check that `unmarshalConfig` returns an exact match of `minimalConfig`. Since `unmarshalConfig` now returns the config with defaults applied, I needed to add a set defaults here to match.